src/main/java/com/google/gerrit/extensions/api/changes/Changes.java

```diff
@@ -74,7 +75,7 @@ ChangeApi id(String project, String branch, String id)
     public abstract List<ChangeInfo> get() throws RestApiException;
 
     public QueryRequest withQuery(String query) {
-      this.query = query;
+      this.query = StandardKeyEncoder.encode(query);
```

Comment:
Is this change backward compatible (i.e. does it work when users of this lib pass already encoded queries like the IntelliJ plugin does)?